modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -412,9 +508,48 @@
 		return calls
 	end
 
+	m.elements.linuxClCompile = function(cfg)
+		local calls = {
+			-- Common - TODO REVIEW LIST
```

Comment:
TODO?

Reply:
Apologies, I've been out for the last 2 weeks. LGTM. Thoughts on a slight change to have 3 tables: 1 for common, one for linux, one for windows, then just join the function tables rather than duplicating?

Reply:
@nickclark2016 good to have your approval, I'll take that bit of feedback onboard and I still need to finish up a couple other things that were reported by @Jarod42, I'll try to find some good time this week

Reply:
@nickclark2016 The problem with having a common table is that the ordering changes and fixing the tests is quite a pain. I think I'll keep it as it is for the moment until I nail down what the common parts should be, then we can do that in another PR?

Reply:
Works for me. Appreciate the leg work on investigating that.